version: "1.0.0"

requirements:
  rmse_position: 0.5
  rmse_tilt: 0.5

main_timer:
  rate: 100 # [Hz]

start_timer:
  rate: 1 # [Hz]

path: [
  # x, y, z, heading
  0.0, 0.0, 1.5, 0.0,
  1.0, 0.2, 1.5, 0.0,
  2.0, -0.2, 1.5, 0.0,
  3.0, 0.2, 1.5, 0.0,
  4.0, -0.2, 1.5, 0.0,
  5.0, 0.2, 1.5, 0.0,
  6.0, -0.2, 1.5, 0.0,
  7.0, 0.2, 1.5, 0.0,
  10.0, 0.0, 1.5, 0.0,
  25.0, 2.5, 1.5, 0.0,
  33.0, 2.5, 1.5, 0.0,
  34.2, -7.72, 2.5, 0.0,
  46.8, -15.72, 2.5, 0.0,
  25.8, -7.72, 8.5, 0.0,
  0.0, 0.0, 1.5, 0.0,
]

relax_heading: true
use_heading: true
stop_at_waypoints: false
